templates/data/prow-periodics-data.yaml

```diff
@@ -21,6 +21,24 @@ templates:
                 mountPath: /etc/config
                 readOnly: true
         jobConfigs:
+          - repoName: kyma-project/eventing-tools
+            jobs:
+              - jobConfig:
+                  name: eventing-tools-autobump
+                  cron: "30 * * * 1-5"
+                  slack_channel: kyma-prow-alerts
+                  image: gcr.io/k8s-prow/generic-autobumper:v20230226-ef2af11076
+                  command: generic-autobumper
+                  args:
+                    - --config=prow/autobump-config/eventing-tools-autobump-config.yaml
+                inheritedConfigs:
+                  local:
+                    - periodic_config
+                    - github_token_mounts
+                  global:
+                    - "pubsub_labels"
+                    - "disable_testgrid"
+                    - extra_refs_test-infra
```

Comment:
You don't need extra_refs if you don't refer to any of the scripts from test-infra. Image should be self contained

Reply:
ill remove it then